### PR TITLE
Measure gas cost of unstake, restake and claim

### PR DESCRIPTION
**Pull request by VladasZ**
Task:
- Measure gas cost of unstake, restake and claim.

### CODE REVIEW

File: model/src/jar_view.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    AccountId,
+};
+
+use crate::{u32::U32, ProductId};
+
+pub type JarIdView = U32;
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+#[serde(crate = "near_sdk::serde")]
+pub struct JarView {
```
Comment (vasyafromrussia) [inline model/src/jar_view.rs:13]: Let's rename file to `jar.rs`? And `withdraw.rs` as well. Name should refer to the meaning of entity I think

File: model/src/u32.rs
Comment (vasyafromrussia) [inline model/src/u32.rs:1]: And this one can be `numbers.rs`

File: model/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+pub use crate::u32::U32;
+
+pub type ProductId = String;
+
+/// Amount of fungible tokens
+pub type TokenAmount = u128;
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Fee {
```
Comment (vasyafromrussia) [inline model/src/lib.rs:19]: This one is about Withdrawal as well

File: integration-tests/src/measure/claim.rs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        .collect();
+
+    dbg!(&map);
+
+    Ok(())
+}
+
+#[ignore]
+#[tokio::test]
+async fn one_claim() -> anyhow::Result<()> {
```
Comment (vasyafromrussia) [inline integration-tests/src/measure/claim.rs:64]: -> single_claim

File: model/src/lib.rs (hunk 2 above)
Comment (VladasZ) [reply model/src/lib.rs]: Fixed

File: model/src/u32.rs
Comment (VladasZ) [reply model/src/u32.rs:1]: Fixed

File: model/src/jar_view.rs (hunk 1 above)
Comment (VladasZ) [reply model/src/jar_view.rs]: Fixed

File: integration-tests/src/measure/claim.rs (hunk 3 above)
Comment (VladasZ) [reply integration-tests/src/measure/claim.rs]: Fixed

**Review by vasyafromrussia — APPROVED** [review]

**Pull request merged**